packages/styled-components/src/sheet/test/Rehydration.test.js

```diff
@@ -23,31 +23,37 @@ describe('outputSheet', () => {
 
     const output = outputSheet(sheet)
       .trim()
-      .split('\n');
-
-    expect(output).toEqual([
-      '.a {}',
-      `${SC_ATTR}.g11[id="idA"]{content:"nameA,"}`,
-      '.b {}',
-      `${SC_ATTR}.g22[id="idB"]{content:"nameB,"}`,
-    ]);
+      .split('/*!sc*/');
+
+    expect(output).toMatchInlineSnapshot(`
+      Array [
+        ".a {}",
+        "
+      data-styled.g11[id=\\"idA\\"]{content:\\"nameA,\\"}",
+        "
+      .b {}",
+        "
+      data-styled.g22[id=\\"idB\\"]{content:\\"nameB,\\"}",
```

Comment:
Why is the output double-escaped now when it wasn't before? That seems like an unwanted change.

Reply:
I think this is just a jest snapshot serialization artifact